plot-api/src/commonMain/kotlin/jetbrains/letsPlot/stat/stat_qq2_line.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022. JetBrains s.r.o.
+ * Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+ */
+
+package jetbrains.letsPlot.stat
+
+import jetbrains.letsPlot.Pos
+import jetbrains.letsPlot.Stat
+import jetbrains.letsPlot.intern.GeomKind
+import jetbrains.letsPlot.intern.Options
+import jetbrains.letsPlot.intern.layer.geom.PathAesthetics
+import jetbrains.letsPlot.intern.layer.geom.PathMapping
+import jetbrains.letsPlot.intern.layer.stat.QQ2LineStatParameters
+import jetbrains.letsPlot.intern.layer.stat.QQ2StatAesthetics
+
+@Suppress("ClassName")
+class statQQ2Line(
+    data: Map<*, *>? = null,
+    geom: jetbrains.letsPlot.intern.layer.GeomOptions = jetbrains.letsPlot.intern.layer.GeomOptions(GeomKind.Q_Q_2_LINE),
```

Comment:
Value should be a specific instance declared in `jetbrains.letsPlot.Geom` object.